dogowner/test_dog_owner.py

```diff
@@ -0,0 +1,97 @@
+from .models import DogOwner
+from django.contrib.auth.models import User
+from django.core.exceptions import ValidationError
+import pytest
+
+EMAIL = 'testuser@gmail.com'
+USERNAME = 'testuser01'
+PASSWORD = 'testpassowrd'
+DOG_NAME = 'kliford'
+FIRST_NAME = 'NEW'
+LAST_NAME = 'USER'
+PHONE_NUMBER = 1234567890
+DOG_RACE = 'lavrador'
+DOG_PICTURE_URL = 'https://www.google.com/'
+DOG_AGE = 10
+DOG_WEIGHT = 6
+DOG_GENDER = 'M'
+
+
+@pytest.fixture
+def dogOwner_01():
```

Comment:
Please rename this fixture per what it does, it should be understood by its name.

Reply:
Fixed.